lang/cs/Org.Apache.REEF.IO.TestClient/Run.cs

```diff
@@ -0,0 +1,51 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System;
+
+namespace Org.Apache.REEF.IO.TestClient
+{
+    /// <summary>
+    /// This purpose of this test is to run tests in Yarn envionment
+    /// 
+    /// </summary>
+    public class Run
```

Comment:
We can easily do something really trivial like:
```c#
public static class Assert
{
   public static void IsTrue(bool condition, string message = null)
  {
        if (!condition)
        {
           Logger.Log("Assert failed; {0}", message);
           throw new AssertFailedException(message); 
        }
  }
}

public class AssertFailedException : Exception
{
         public AssertFailedException(string message) : base(message) {}
}
```
This would make failures much clearer and making this proper unittests easy, once REEF-729 (move to xUnit) is completed.